switchdev_utils/config-service/README.md

```diff
@@ -0,0 +1,55 @@
+# Switchdev Network Configuration Service 
+## Description
+This folder contains SR-IOV, OVS and network configuration service for switchdev and VF-LAG  
+
+The configuration listed below shall be performed as part of a systemd script triggered during machine boot and IFCFG files for network configuration and each configuration step is optionally performed on specified interfaces via configuration file:  
+- Create VFs  
+- Move to switchdev  
+- Set VF trust for all VFs  
+- Create bond over PFs  
+- Create OVS bridges  
+- Enable hw-offload in OVS  
+- Connect bond to bridge  
+
+## Prerequisites
+The below is assumed to be installed/configured on the host
+- NICs Firmware up to date  (as bundled with MLNX_OFED 5.2)
+- (MLNX_OFED 5.2 or above ) or (mft 4.16 or above)
```

Comment:
we discussed using mstflint IMO no ?

Reply:
Is there a way to do vf trust using mstflint ?! @adrianchiris